### ISSUE TITLE
Wdio couldn't create a new chromedriver session (ERROR @wdio/runner: Error)

### ISSUE THREAD

**Issue by Trueside** [webdriverio/webdriverio#6980]
Hello, I'm trying to run tests in docker container (selenium/standalone-chrome) and get some '@wdio/runner: Error' without any context. Pls, help 😥

**Environment (please complete the following information):**
 - **WebdriverIO version:** 7.5.7
 - **Mode:** WDIO Testrunner
 - **If WDIO Testrunner, running sync/async:** async
 - **Node.js version:** v14.7.0
 - **NPM version:** 6.14.7
 - **Browser name and version:** 84.0
 - **Platform name and version:** Ubuntu
 - **Additional wdio packages used (if applicable):** 
 - package.json
```
  "dependencies": {
    "@wdio/cli": "^7.5.7"
  },
  "devDependencies": {
    "@wdio/devtools-service": "^7.5.7",
    "@wdio/local-runner": "^7.5.7",
    "@wdio/mocha-framework": "^7.5.3",
    "@wdio/selenium-standalone-service": "^7.5.7",
    "@wdio/spec-reporter": "^7.5.7",
    "expect-webdriverio": "^2.0.1",
    "ts-node": "^9.1.1",
    "typescript": "^4.2.4"
  }
}
```

**Config of WebdriverIO**

```
export const config: WebdriverIO.Config = {
    runner: 'local',
    automationProtocol: 'devtools',
    specs: [
    ],
    exclude: [
        // 'path/to/excluded/files'
    ],
    maxInstances: 2,
    capabilities: [{
        'wdio:devtoolsOptions': {
            ignoreDefaultArgs: true
        },
        'goog:chromeOptions': {
            args: [
                '--window-size=880,710',
                '--mute-audio',
                '--ash-disable-touch-exploration-mode ',
                '--disable-extensions'

                // /* headless mode
                '--headless',
                '--disable-infobars',
                '--disable-gpu',
                '--custom-launcher-page',
                '--extensions-on-chrome-urls',
                '--extensions-not-webstore',
                // */
            ],
        },
            maxInstances: 3,
            browserName: 'chrome',
            platformName: 'Windows 10', // OS platform
            acceptInsecureCerts: true
        }],
    logLevel: 'info',
    bail: 0,
    baseUrl: 'https://app-stage.com/',
    waitforTimeout: 5999,
    connectionRetryTimeout: 7555,
    connectionRetryCount: 3,
    services: [
        // 'selenium-standalone', 
        'devtools'
    ],
    framework: 'mocha',
    mochaOpts: {
        ui: 'bdd',
        timeout: 498877,
        // bail: true,
        // allowUncaught: true,
        // checkLeaks: true,
        // fullTrace: true
    },
    autoCompileOpts: {
        autoCompile: true,
        // see https://github.com/TypeStrong/ts-node#cli-and-programmatic-options
        // for all available options
        tsNodeOpts: {
            transpileOnly: true,
            project: 'tsconfig.json'
        },
    },
}
```

**Describe the bug**
I create container from 'selenium/standalone-chrome' image and try to run tests inside.
This configuration works on my local machine without docker, but when I try to run tests inside docker container it fails.

**To Reproduce**
create docker container from `selenium/standalone-chrome` image
try to run a test inside the container.

**Expected behavior**
Tests run.
 
**Log**

```
2021-06-06T15:39:20.998Z INFO @wdio/cli:launcher: Run onPrepare hook
2021-06-06T15:39:21.000Z INFO @wdio/cli:launcher: Run onWorkerStart hook
2021-06-06T15:39:21.001Z INFO @wdio/local-runner: Start worker 0-0 with arg: ./wdio.conf.ts
[0-0] 2021-06-06T15:39:21.499Z INFO @wdio/local-runner: Run worker command: run
[0-0] RUNNING in chrome - /test/specs/deploy/long.deploy.spec.ts
[0-0] 2021-06-06T15:39:22.856Z INFO devtools:puppeteer: Initiate new session using the DevTools protocol
[0-0] 2021-06-06T15:39:22.857Z INFO devtools: Launch Google Chrome with flags: --window-position=0,0 --window-size=1200,900 --window-size=820,710 --mute-audio --ash-disable-touch-exploration-mode  --headless --disable-infobars --disable-gpu --custom-launcher-page --extensions-on-chrome-urls --extensions-not-webstore
[0-0] 2021-06-06T15:39:23.096Z ERROR @wdio/runner: Error:
[0-0]     at new LauncherError (/home/circleci/app/e2e/node_modules/devtools/node_modules/chrome-launcher/src/utils.ts:31:18)
[0-0]     at new ChromePathNotSetError (/home/circleci/app/e2e/node_modules/devtools/node_modules/chrome-launcher/dist/utils.js:33:9)
[0-0]     at Object.linux (/home/circleci/app/e2e/node_modules/devtools/node_modules/chrome-launcher/src/chrome-finder.ts:153:11)
[0-0]     at Function.getFirstInstallation (/home/circleci/app/e2e/node_modules/devtools/node_modules/chrome-launcher/src/chrome-launcher.ts:177:61)
[0-0]     at Launcher.launch (/home/circleci/app/e2e/node_modules/devtools/node_modules/chrome-launcher/src/chrome-launcher.ts:223:37)
[0-0]     at Object.launch (/home/circleci/app/e2e/node_modules/devtools/node_modules/chrome-launcher/src/chrome-launcher.ts:71:18)
[0-0]     at launchChrome (/home/circleci/app/e2e/node_modules/devtools/build/launcher.js:65:44)
[0-0]     at Object.launch [as default] (/home/circleci/app/e2e/node_modules/devtools/build/launcher.js:164:16)
[0-0]     at Function.newSession (/home/circleci/app/e2e/node_modules/devtools/build/index.js:50:49)
[0-0]     at Object.remote (/home/circleci/app/e2e/node_modules/webdriverio/build/index.js:67:43)
[0-0] FAILED in chrome - /test/specs/deploy/long.deploy.spec.ts
2021-06-06T15:39:23.211Z INFO @wdio/cli:launcher: Run onComplete hook

Spec Files:      0 passed, 1 failed, 1 total (100% completed) in 00:00:02

2021-06-06T15:39:23.212Z INFO @wdio/local-runner: Shutting down spawned worker
2021-06-06T15:39:23.463Z INFO @wdio/local-runner: Waiting for 0 to shut down gracefully
2021-06-06T15:39:23.464Z INFO @wdio/local-runner: shutting down
```

**Additional context**
Test example
```
import LoginPage from '../../pageobjects/login.page';
import { testData } from '../../data/n.login.data'

describe('NEGATIVE LOGIN', async () => {

  beforeEach(async () => {
    await LoginPage.open();
  })

  for (const tcase of Object.values(testData)) {
    it(`${tcase.INFO}`, async () => {
      await LoginPage.login(
        tcase.EMAIL,
        tcase.PASSWORD,
        undefined, true
      );
      await LoginPage.findError(tcase.ERR);
    })
  }
})
```

Dockerfile
```
FROM selenium/standalone-chrome:84.0
RUN curl -sL https://deb.nodesource.com/setup_14.x | sudo -E bash -
RUN sudo apt-get install -y git nodejs g++ build-essential
RUN sudo npm install yarn -g
```

> Please use the correct markdown

**Comment by christian-bromann** [comment]
Not sure if this is actually the reason but there is a missing `,` after `'--disable-extensions'`.

After fixing it I was able to run tests in this environment.

**Comment by Trueside** [comment]
@christian-bromann  Hello, ofc it's not, but if u are not interested in my problem, can u pls give the cue where I should look for solution, because error 'ERROR @wdio/runner: Error' tells nothing about the problem.

**Comment by christian-bromann** [comment]
> but if u are not interested in my problem

I am not sure what you want to suggest here. I looked into your issue and said that I was able to run tests. My Dockerfile looks as follows:

```Dockerfile
FROM selenium/standalone-chrome:84.0
RUN curl -sL https://deb.nodesource.com/setup_14.x | sudo -E bash -
RUN sudo apt-get install -y git nodejs g++ build-essential
RUN sudo npm install yarn -g
CMD ["sh", "-c", "tail -f /dev/null"]
```

I build and ran it and was able to run tests.

> 'ERROR @wdio/runner: Error' tells nothing about the problem.

This is everything we get back from the `chrome-launcher` package. To me it indicates that there is a problem launching Chrome which is weird because I am perfectly capable doing this in the given environment.

So what can I say, I tried everything I can, but then getting accused of not trying to help is really un-polite from you saying. Just because an issue is being closed it doesn't mean anyone discards your problem, it just means that this is unlikely a bug that can be fixed in WebdriverIO and more an environment issue. I would have continued helping you out even after an issue is closed.
